lms/templates/courseware/legacy_instructor_dashboard.html

```diff
@@ -329,8 +329,14 @@ <h2 class="navbar">[ <a href="#" onclick="goto('Grades');" class="${modeflag.get
     </div>
     % endif
 
-    <input type="submit" name="action" value="List enrolled students" class="${'is-disabled' if disable_buttons else ''}" aria-disabled="${'true' if disable_buttons else 'false'}">
-    <input type="submit" name="action" value="List students who may enroll but may not have yet signed up" class="${'is-disabled' if disable_buttons else ''}" aria-disabled="${'true' if disable_buttons else 'false'}" >
+    <p>
+      <input type="submit" name="action" value="List enrolled students" class="${'is-disabled' if disable_buttons else ''}" aria-disabled="${'true' if disable_buttons else 'false'}">
```

Comment:
As per the discussion on https://openedx.atlassian.net/browse/OSPR-171 we're not planning to port this. Can you please add a message that direct users to the Data Download section's feature to "Download Profile Information as a CSV"